### PR TITLE
Updates new GA4 cookies to privacy doc

### PR DESCRIPTION
**Pull request by nyashamaphosa**
## Status

* Current Status: Ready for (front-end / tech) review
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2329

## What's changed?

*updates privacy doc with new GA4 cookies*

Matching copy from console cookie storage tab
Screenshot 2023-04-14 at 15 05 14

### CODE REVIEW

**Comment by dcorking** [comment]
Is your plan to add the rails session cookie and ratings cookie to this PR?

**Comment by nyashamaphosa** [comment]
> Is your plan to add the rails session cookie and ratings cookie to this PR?

Section 10 appears to describe purpose of session cookies so i thought i leave that alone. The ratings cookies does not appear to be present/listed in the storage tab

**Comment by dcorking** [comment]
Thanks for being patient re an answer to your 2 questions.

1. I think because PHP Session ID is listed in the 'Necessary and Functional Cookies' section we should add the Rails session cookie to the same list.

2. to get a ratings cookie you need to
a. log in (to staging in this example)
b. visit a resource such as https://staging.teachcomputing.org/curriculum/key-stage-2/computing-systems-and-networks-connecting-computers
c. click the thumbs up or thumbs down rating.

Does that help?

**Comment by nyashamaphosa** [comment]
> 

Yes that helps thank you

File: app/views/pages/privacy.html.erb [hunk 1]
```diff
@@ -302,13 +302,23 @@
...
+        <td class="govuk-table__cell">SSESS
+        <br>PHPSESSID</td>
+        <td class="govuk-table__cell">Used by the website to
+        identify a user that is
+        logged in</td>
+        </tr>
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header ncce-table__header" scope="row">Ratings</th>
+          <td class="govuk-table__cell">ratings</td>
+          <td class="govuk-table__cell">Used by the website to identify a user rating on a course</td>
```
Comment (dcorking) [inline app/views/pages/privacy.html.erb:321]:
```suggestion
          <td class="govuk-table__cell">Used by the website to identify a user rating on a teaching resource</td>
```

Pls also fix the indentation of lines 310 to 316.

File: app/views/pages/privacy.html.erb [hunk 2]
```diff
@@ -430,7 +440,7 @@
           <th class="govuk-table__header ncce-table__header" scope="row">
             Universal
             analytics (Google)</th>
```
Comment (dcorking) [inline app/views/pages/privacy.html.erb:442]:
```suggestion
            Google
            Analytics</th>
```

File: app/views/pages/privacy.html.erb [hunk 3]
```diff
@@ -430,7 +440,7 @@
...
-          <td class="govuk-table__cell">_ga, _gat, _gid</td>
+          <td class="govuk-table__cell">_ga, _gat, _gid, _gcl_au, _ga_GRKX55937M _rollupGA, _rollupGA_gid</td>
```
Comment (dcorking) [inline app/views/pages/privacy.html.erb:443]:
```suggestion
          <td class="govuk-table__cell">_ga, _gat, _gid, _gcl_au, _ga_GRKX55937M, _rollupGA, _rollupGA_gid</td>
```

**Review by dcorking — APPROVED** [review]
Pls can you make the tweaks that I suggest below, and then ship it.
🚂

**Comment by nyashamaphosa** [comment]
> Pls can you make the tweaks that I suggest below, and then ship it. 🚂

@dcorking resolved thank you

**Pull request merged**